lib/google/gax/settings.rb

```diff
@@ -400,9 +400,11 @@ def upper_camel_to_lower_underscore(string)
     #   {
     #     "interfaces": {
     #       "google.fake.v1.ServiceName": {
-    #         "retry_codes": {
-    #           "idempotent": ["UNAVAILABLE", "DEADLINE_EXCEEDED"],
-    #           "non_idempotent": []
+    #         "retry_codes_def": {
```

Comment:
We decided to remove 'retry_codes_def' from all json files. So I'm closing this PR.